packages/keystone/access.js

```diff
@@ -140,6 +142,20 @@ function isFilteringBy (where, fields) {
     return false
 }
 
+function isSpecificQuery (queryName) {
```

Comment:
This name could be misleading. Is it possible to choose something more intuitive?